Src/WitsmlExplorer.Api/appsettings.Development.json

```diff
@@ -1,4 +1,8 @@
 {
+"MongoDb": {
```

Comment:
Remove. Any settings needed to run the project locally should be put in the mysettings.json file as described in https://github.com/equinor/witsml-explorer/blob/main/CONTRIBUTING.md#set-up-development-environment

Reply:
dev appsettings added to gitignore to avoid commit of local dev settings in future

Reply:
appsettings.development.json targets the deployed dev environment (part of the dev-test-prod deployment scheme). It is not supposed to be in .gitignore. As I said in my previous comment, the "mysettings.json" file, which is in .gitignore, is supposed to be used for local settings.